### PR TITLE
SQLTable: add row filters

### LINKED ISSUE #351: Allow column filtering in table
Could replace `fixed-data-table` with http://adazzle.github.io/react-data-grid/examples.html#/filterable-grid

### PR DESCRIPTION
**Pull request by n-riesco**
* Replaced `fixed-data-table` with `react-data-grid`

* Implemented row filters

Closes #351 

---

@tarzzz Lately, I've been asking you to review only PRs for the backend. Would you like to review this PR?

@jackparmer Would you take this PR for a spin and see if you'd like to change anything?

### CODE REVIEW

**Comment by jackparmer** [comment]
@n-riesco I'm having trouble running the artifact. Could you share a screencast?

**Comment by n-riesco** [comment]
@jackparmer could you describe what the trouble was? was it hard to discover? should I move the message `(click on the header to toggle the row filter)` to the top? were you using a phone?

peek 2018-03-02 08-17

**Comment by jackparmer** [comment]
Thanks! I think this looks good.

File: static/index.html [hunk 1]
```diff
@@ -5,6 +5,8 @@
     <meta charset="utf-8">
     <title>Falcon SQL Client</title>
 
+    <link rel="stylesheet" type="text/css" href="static/css/falcon.css"></link>
```
Comment (shannonlal) [inline static/index.html]: Probably too late to change; however, is there a reason we have separate css files and not using something webpack to bundle this.  Again not important just an observation

Comment (shannonlal) [reply static/index.html]: @n-riesco Looks good as well 💃

Comment (n-riesco) [reply static/index.html]: @shannonlal 

> Probably too late to change; however, is there a reason we have separate css files and not using something webpack to bundle this. Again not important just an observation

This is a temporary solution. I'd like to come back to packing CSS with webpack and possibly using http://lesscss.org/ to define a some sort of theme to be used by all components.

Comment (jackparmer) [reply static/index.html]: >  I'd like to come back to packing CSS with webpack and possibly using http://lesscss.org/ to define a some sort of theme to be used by all components.

I've heard good things about [Tachyons](http://tachyons.io/), which seems to be what [SQLPad is using](https://github.com/rickbergfalk/sqlpad/pull/274). Should be easy enough to harmonize something like this with plotly's [styleguide](https://brand.plot.ly/).

Comment (n-riesco) [reply static/index.html]: Jack, as usual, your collections of links are gold! :)

**Pull request merged**